observation_portal/observations/serializers.py

```diff
@@ -30,6 +30,7 @@ class ConfigurationStatusSerializer(serializers.ModelSerializer):
     instrument_name = serializers.CharField(required=False)
     guide_camera_name = serializers.CharField(required=False)
     end = serializers.DateTimeField(required=False)
+    start_exposures = serializers.DateTimeField(required=False)
```

Comment:
I named the update field this for now, but was thinking maybe it should be more descriptive of what it is, like `start_exposures_at` or `exposures_start_at`.